deployment/tasks/build.yml

```diff
@@ -19,5 +19,8 @@
 - name: Compile JavaScript
   command: npm run build-js chdir={{ build_dir }}
 
+- name: Build assets
```

Comment:
You need to switch the order because `build-assets` creates the `_js` directory that `build-js` uses.